[cutlass-3] Add cutlass key for fbcode and OSS

henrylhtsang opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #153081

Differential Revision: [D74337959](https://our.internmc.facebook.com/intern/diff/D74337959/)

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @chenyang78 @kadeng @muchulee8 @amjames @chauhang @aakhundov

henrylhtsang committed:
[cutlass-3] Add cutlass key for fbcode



Differential Revision: [D74337959](https://our.internmc.facebook.com/intern/diff/D74337959/)

[ghstack-poisoned]

henrylhtsang committed:
Update on "[cutlass-3] Add cutlass key for fbcode"


Differential Revision: [D74337959](https://our.internmc.facebook.com/intern/diff/D74337959/)

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

henrylhtsang committed:
Update on "[cutlass-3] Add cutlass key for fbcode"


Differential Revision: [D74337959](https://our.internmc.facebook.com/intern/diff/D74337959/)

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

henrylhtsang committed:
Update on "[cutlass-3] Add cutlass key for fbcode"


Differential Revision: [D74337959](https://our.internmc.facebook.com/intern/diff/D74337959/)

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

henrylhtsang committed:
Update on "[cutlass-3] Add cutlass key for fbcode"


Differential Revision: [D74337959](https://our.internmc.facebook.com/intern/diff/D74337959/)

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

henrylhtsang committed:
Update on "[cutlass-3] Add cutlass key for fbcode and OSS"


Differential Revision: [D74337959](https://our.internmc.facebook.com/intern/diff/D74337959/)

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

drisspg reviewed (approved):
I think this is fine, cc @oulgen

henrylhtsang committed:
Update on "[cutlass-3] Add cutlass key for fbcode and OSS"


Differential Revision: [D74337959](https://our.internmc.facebook.com/intern/diff/D74337959/)

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

henrylhtsang committed:
Update on "[cutlass-3] Add cutlass key for fbcode and OSS"


Differential Revision: [D74337959](https://our.internmc.facebook.com/intern/diff/D74337959/)

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

henrylhtsang committed:
Update on "[cutlass-3] Add cutlass key for fbcode and OSS"


Differential Revision: [D74337959](https://our.internmc.facebook.com/intern/diff/D74337959/)

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

henrylhtsang commented:
@pytorchbot merge -i
